addon-test-support/setup-global-a11y-hooks.ts

```diff
@@ -24,6 +24,10 @@ type HelperName =
   | 'typeIn'
   | 'visit';
 
+interface HookUnregister {
+  unregister: () => void;
+}
```

Comment:
Considering the changes in [PR #1344](https://github.com/emberjs/ember-test-helpers/pull/1344), a 2.9.4 release of @ember/test-helpers would allow us to just import `registerHook` and `HookUnregister` as official public APIs. Is there a reason why we can't just wait for that release before making these changes? Seems like duplicative work.

Reply:
Right now, I'm trying to migrate an addon to typescript; this PR unblocks that as there is an issue with a reference to ember__test-helpers. Being that the code is already written and ready for merging.  Seems like the duplicative work has been done.

Do we know when the PR referenced above will be released? @drewlee

Reply:
That would be a question for @chriskrycho, who maintains the `@ember/test-helpers` releases.

Reply:
To be specific, the duplicative work would entail removing the inline `HookUnregister` type that's being introduced with this change. Just seems more efficient to incorporate these changes wholesale in accommodating a 2.9.4 release of `@ember/test-helpers`. We've been aware of the TypeScript bug (see issue #498 ) and is one of the reasons why the changes were carried out in [PR #1344](https://github.com/emberjs/ember-test-helpers/pull/1344).

Reply:
I see.  I pushed this PR at the suggestion of @chriskrycho.  Chris, is there a reason why we cant just release `@ember/test-helpers` and then update this addon?